### PR TITLE
Remove self from maintainer-list.nix

### PR DESCRIPTION
**Pull request by shanemikel**
It has been a while!  I'm sorry to say I do not expect to contribute in the immediate future, as I have not used Nix for several years.

I don't believe I have my name on any packages any longer.  I had contributed updates to one package: htmldoc, which has been adopted by `michaelrsweet`.  I was also the original author of the script in `maintainers/scripts/nix-diff.sh`, but it doesn't have an official owner.

Is there anything else I can fix other than removing my name from the list?

### CODE REVIEW

**Comment by ajs124** [comment]
You're still listed in `pkgs/tools/typesetting/htmldoc/default.nix`. Can you also remove yourself from that?

**Comment by ajs124** [comment]
We could ask some people that last worked on the package if they want to take over maintainership, e.g. @FliegendeWurst or @jiegec

**Comment by Mindavi** [comment]
At least remove yourself there so this can be merged if nobody responds. It causes an eval error now

**Comment by bjornfor** [comment]
Please rebase: remove the merge commit and order the commits so that you remove yourself from packages first, then remove yourself from the maintainers list. (Opposite order breaks eval for the intermediate commit / breaks git bisect.)

**Review by bjornfor — APPROVED** [review]

**Pull request merged**
